knowledge-base/kb-panelbar-remove-mouseover-and-selected-overlay-child-item.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Remove MouseOver and Selected Overlay on Child Item
+description: Avoid mouse enter and selection backgrounds for child RadPanelBarItem when it selected or hovered
+type: how-to
+page_title: Clear MouseOver and Selected Backgrounds from Child Item
+slug: kb-panelbar-remove-mouseover-and-selected-overlay-child-item
+position: 0
+tags: hover,selection,overlay,background
+ticketid: 1401889
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>2019.1.220</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>RadPanelBar for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+How to remove mouse over and selected background colors of a child RadPanelBarItem when it is hovered or selected.
+
+## Solution
+
+Extract and modify the content of the Template property of RadPanelBarItem.
+
+__To remove the mouse over overlay:__
+
+1. Extract the Style of the RadPanelBarItem. 
+2. Find the Setter that sets the Template property.
+3. Find the VisualState with x:Name="MouseOver" in the template and remove it.
+4. Find the Border element with x:Name="MouseOverVisual" and remove it as well.
+
+The following elements are the ones that should be removed from the template:
+
+#### __[XAML]__
+{{region kb-panelbar-remove-mouseover-and-selected-overlay-child-item-0}}
+	<VisualState x:Name="MouseOver">
+		<Storyboard>
+			<DoubleAnimation Duration="0:0:0.2" To="1" Storyboard.TargetProperty="Opacity" Storyboard.TargetName="MouseOverVisual"/>
+		</Storyboard>
+	</VisualState>
+	<!---------------------------->
+	<Border x:Name="MouseOverVisual" BorderBrush="#FFFFC92B" BorderThickness="1" CornerRadius="1" Opacity="0">
+		<Border.Background>
+			<LinearGradientBrush EndPoint="0.5,1" StartPoint="0.5,0">
+				<GradientStop Color="#FFFFFBA3" Offset="1"/>
+				<GradientStop Color="#FFFFFBDA" Offset="0"/>
+			</LinearGradientBrush>
+		</Border.Background>
+		<Border BorderBrush="White" BorderThickness="1" Background="{x:Null}" CornerRadius="0"/>
+	</Border>
+{{endregion}}
+
+__To remove the selected overlay:__
+
+1. Extract the Style of the RadPanelBarItem. 
+2. Find the Setter that sets the Template property.
```

Comment:
Again we can specify in this step or in a separate step that the RadPanelBarItemTemplate needs to be located.